### ISSUE TITLE
Not possible to require this lib in Racket 7.0

### ISSUE THREAD

**Issue by dkvasnicka** [stchang/ftree#3]
```
▶ racket --lib ftree
open-input-file: cannot open module file
  module path: (lib ftree)
  path: /Users/danielkvasnicka/Library/Racket/7.0/pkgs/ftree/main.rkt
  system error: no such file or directory; rktio_err=3
  context...:
   maybe-raise-missing-module
   default-load-handler
   standard-module-name-resolver
   module-path-index-resolve
   [repeats 1 more time]
   module-declared?
```

```
▶ racket --lib intervaltree
standard-module-name-resolver: collection not found
  for module path: (lib "intervaltree")
  collection: "intervaltree"
  in collection directories:
   /Users/danielkvasnicka/Library/Racket/7.0/collects
   /Applications/Racket v7.0/collects
   ... [199 additional linked and package directories]
  context...:
   show-collection-err
   standard-module-name-resolver
   module-path-index-resolve
   [repeats 1 more time]
   module-declared?
```

Can be worked around by:
`(require (file "~/Library/Racket/7.0/pkgs/ftree/intervaltree/main.rkt"))`

**Comment by MichaelMMacLeod** [comment]
#4 should address this problem

**Comment by stchang** [comment]
fixed by #4
